### DISCUSSION TITLE
blog/writing/2024/06/01/advice-to-young-people/

### DISCUSSION THREAD

**Discussion by giscus** [jxnl/blog#68] [General]
# blog/writing/2024/06/01/advice-to-young-people/

Notes about my hobbies and other things I find interesting.

https://jxnl.github.io/blog/writing/2024/06/01/advice-to-young-people/

**Comment by rifkyzena** [comment]
This is really good. I don't know why nobody made any comments on this yet.

**Comment by AnnthomyGILLES** [comment]
I'm glad that I came across your writings, and specifically this one.

Thank you 🙏🏿

**Comment by VanessaCiretose** [comment]
Amazing read, I felt the same way after I had my most recent traumatic event. Spending a lot of time fixed me. I'll be sure to come back to this to see if there is anything I can learn again.

**Reply by jxnl** [reply]
"you were just a child"

**Comment by ritvikvarghese** [comment]
lovely tuesday morning read, thanks Jason :)

**Comment by ChiehYunChen** [comment]
Love your story, especially the self-worth section!
Thank you for sharing the insight, Jason.

**Comment by raghunandan-r** [comment]
not young anymore, but appreciate the post.

**Comment by adheeb2** [comment]
I just loved your way of thinking mate. As a 23 year old myself, i have always had a feeling that i missed my chances and that i should have taken every single decisions perfectly. But your blog has just made my day. The part where you explained about "You are a vessel, a medium for the work, but you are not the work itself." It really resonated with me. I felt like my work described me and that if i failed even in the tiny bits of my work,then i have failed myself. I genuinely thank you for creating this post mate. I really wish the best for you💜

**Comment by 0xrsydn** [comment]
didnt expect the writing to be this personal, appreciate and love ur writing!

**Comment by sanjeed5** [comment]
great read. thanks for sharing! I owe a lot of my success to your writings!

**Comment by captaindpt** [comment]
Wow. Great stuff

**Comment by 0yzmo** [comment]
Thank you for writing this, Jason. This is actually the best article I've read in years.

**Comment by omid-web** [comment]
very touching article, it changed my perception of money and its value overt time.
as someone who grew up in poverty and avid gamer, I also save potions but at end just collects interest that doesnt get fully utilized.

**Comment by will-r-wang** [comment]
I love this

**Comment by enkeejunior1** [comment]
My first year of PhD was not successful, and I ruined my academnic reputation, research, relationshp, etc. But your post helped me feel like I can start again and move in a better direction.

I hope I can come back to this post someday with better version of myself. Thank you for writing this.

**Comment by Aurora-zyy** [comment]
Thanks for writing it, Jason from several years ago! "We are not our work." is a great mindset. You said we were vessel — I looked at it more like people are mantles and every work have potential to  be dust or diamond. We made decisions if we want to suffer from the intense feeling and the stress of sharpening thinking around our past experience to form that extreme. You clearly made the decision so now the article is a true diamond. Tough and brilliant. Lit up many. Long-tail effect. Hope to see more of your writing!

**Comment by jkzhang7** [comment]
Wow what a hidden gem! I wish I had read this much earlier!

**Comment by jmeisburg-chwy** [comment]
Great share! Appreciate the words of wisdom.

**Comment by jeraldine-t** [comment]
“Be someone people want to work with”. what a powerful statement… wish I had come across this piece of advice earlier but it’s never too late!

**Comment by sahaavi** [comment]
This is gold!
